client/src/GettingStarted/GettingStarted.js

```diff
@@ -32,7 +32,7 @@ export function GettingStarted() {
     }
 
     getUser()
-  }, [])
+  }, [makeRequest])
```

Comment:
@katelovescode This causes infinite requests to be sent to `/api/v1/profile`. You can observe it by inspecting the Network tab.

Reply:
Yeah, I saw this, but when I removed the dependency it didn't fix it. O_O Maybe I'm missing something.  Will look again tomorrow.

Reply:
I was mistaken; apparently it worked this morning 🤷